jars/settings.py

```diff
@@ -89,7 +89,7 @@
     'DEFAULT_AUTHENTICATION_CLASSES': (
         # 'rest_framework.authentication.BasicAuthentication',
         'rest_framework.authentication.TokenAuthentication',
-        'oauth2_provider.ext.rest_framework.OAuth2Authentication',
+        'oauth2_provider.contrib.rest_framework.OAuth2Authentication',
```

Comment:
this needs to stay what it was before

Reply:
The Travis check fails if I put it as before. Shall I revert it still ?

Reply:
yes, we've talked about this, didn't we? it needs it on production but the tests fail. we have to fix that eventually.